### ISSUE TITLE
scpOptions needs to be a list/array of strings

### ISSUE THREAD

**Issue by benjohnson2001** [edvin/fxlauncher#70]
in DeployAppTask of the plugin I'm not able to specify a private key because both "-i" and "/my/privatekey/location" are needed but scpOptions is just a single string

```
...

    def params = ["scp"]
    if (fxlauncher.scpOptions) params.add(fxlauncher.scpOptions)
    params.addAll("-r", fxlauncher.resolveWorkingDirectory().toString() + '/.', fxlauncher.deployTarget)

...
```

**Comment by edvin** [comment]
Sorry about that, I have changed `scpOptions` to be `List<String>` now. I will release a new version of the plugin next week.

**Comment by edvin** [comment]
I just released version 1.0.16 to Maven Central now :)

**Comment by benjohnson2001** [comment]
thank you!! :-)
